.github/workflows/wheel_tests_and_release.yml

```diff
@@ -59,18 +59,15 @@ jobs:
         run: pipx run twine check --strict dist/*
 
       # Ensure that a compromised twine couldn't have altered the distributions
-      # Required to resolve sdist and wheels separately
-      - name: Verify sdist artifact attestation
+      - name: Verify artifact attestation
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: gh attestation verify dist/scikit_image-*.tar.gz --repo ${{ github.repository }}
-
-      # This needs to be fixed, see
-      # https://github.com/scikit-image/scikit-image/issues/7443
-      # - name: Verify wheel artifact attestation
-      #   env:
-      #     GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      #   run: gh attestation verify dist/scikit_image-*.whl --repo ${{ github.repository }}
+        shell: bash
+        run: |
+          for artifact in dist/*; do
+              echo "# ${artifact}"
```

Comment:
@stefanv After you reviewed I added this echo here as in CI things are much less verbose by default, so it can be somewhat nice to see the files actually get printed along with their verification for visual log debugging.

Reply:
Good idea!